src/main/java/homework_4/custom_file_reader/Main.java

```diff
@@ -0,0 +1,15 @@
+package homework_4.custom_file_reader;
+
+public class Main {
+    public static void main(String[] args) {
+        new CustomFileReader("file1.txt").run();
```

Comment:
Doesn't look like expected behaviour, and commented-out lines of code are a code smell
![image](https://user-images.githubusercontent.com/58872509/134029061-9abc9607-becc-4389-a80d-9078fdb26dce.png)

Reply:
It seems does not work 
![image](https://user-images.githubusercontent.com/58872509/134029238-6a4c1dc6-565e-41c2-a318-cdcfb677ecc1.png)

Reply:
Updated. My bad. It should be file.txt, sorry for comments.